authN-authZ/auth-istio/README.md

```diff
@@ -221,31 +221,31 @@ The user management is done via Keycloak and the configuration steps look like t
 
 1. Create a new realm named `chatqna` within Keycloak.
 
-<img src="./docs/create_realm.png" width="600" height="300">
+   ![create realm](./docs/create_realm.png)
 
-2. Create a new client called `chatqna` and set `Client authentication` to 'On'. Set "http://chatqna-ui.com:${INGRESS_PORT}/*" in the `Valid redirect URIs` part. Note that `INGRESS_PORT` and `INGRESS_HOST` shall be exported following the guide [here](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports). Under the Credentials tab you will now be able to locate `<your client's secret>`, which will be used in the oauth2-proxy configs.
+2. Create a new client called `chatqna` and set `Client authentication` to 'On'. Set "http://chatqna-ui.com:${INGRESS_PORT}/\*" in the `Valid redirect URIs` part. Note that `INGRESS_PORT` and `INGRESS_HOST` shall be exported following the guide [here](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports). Under the Credentials tab you will now be able to locate `<your client's secret>`, which will be used in the oauth2-proxy configs.
```

Comment:
User need to set `http://chatqna-ui.com:${INGRESS_PORT}/*` in the field. Suggest to remove the change.

Reply:
Line is restored (change removed)